includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -316,7 +316,12 @@ public static function subscription_pricing_fields() {
 				?>
 			</label>
 			<span class="wrap">
-				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php echo esc_attr_x( 'e.g. 5.90', 'example price', 'woocommerce-subscriptions' ); ?>" step="any" min="0" value="<?php echo esc_attr( wc_format_localized_price( $chosen_price ) ); ?>" />
+				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="
+				<?php
+				// translators: %s the formatted example price value.
+				echo esc_attr( sprintf( 'e.g. %s', wc_format_localized_price( '5.90' ) ) );
+				?>
```

Comment:
There's a couple of issues with the code here: 

1. All the whitespace after `placeholder="` and `<?php` is being rendered in the HTML. 
2. This should be translated like before. 

```php
// translators: %s the formatted example price value.
echo sprintf( esc_attr_x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), wc_format_localized_price( '5.90' ) );
```

Reply:
Fixed on https://github.com/Automattic/woocommerce-subscriptions-core/pull/807/commits/0aaa1807824b07a13e6b80bf802894e674ab8792.